iceoryx_binding_c/include/iceoryx_binding_c/subscriber.h

```diff
@@ -24,22 +25,50 @@
 /// @brief Subscriber handle
 typedef struct cpp2c_Subscriber* iox_sub_t;
 
+/// @brief options to be set for a subscriber
+typedef struct
+{
+    // size of the history chunk queue
+    uint64_t queueCapacity;
```

Comment:
The struct members should be default initialized with zero or nullptr. Applies also for all other structs in this PR.

Reply:
Unfortunately not possible in C. This is why @MatthiasKillat implemented the *_options_init methods.

Reply:
Impossible in C.